components/navbar/ProfileDropdown.vue

```diff
@@ -102,3 +103,15 @@ const toggleLanguageMenu = () => {
   languageDropdown.value?.toggle()
 }
 </script>
+
+<style lang="scss" scoped>
+.menu-mt-0 {
+  :deep(.o-drop__menu) {
+    margin-top: 0;
+  }
+}
+
+.navbar-height {
+  height: 4.5rem;
+}
```

Comment:
fixed height, what if we want to change navbar height?

Reply:
yes but...
https://github.com/kodadot/nft-gallery/pull/6430
see `components/Navar` line 359
```vue
<style lang="scss" scoped>
:deep(.navbar-explore) {
  .navbar-item {
    height: 4.5rem;
  }
...


}
</style>
```

Reply:
i actually replaced the fixed height with margin (my-4)